### ISSUE TITLE
Global numberings

### ISSUE THREAD

**Issue by termi-official** [Ferrite-FEM/FerriteViz.jl#11]
It can be quite helpful to visualize the numbering of different things for debugging purposes

* [ ] Vertex numbering 
* [x] Node numbering (#20)
* [ ] Edge numbering, if existent
* [ ] Face numbering, if existent
* [x] Cell numbering (#20)
* [ ] DoF numbering

as well as having their positions visualized

**Comment by termi-official** [comment]
I would keep it modular and add methods like `add_...` or `print_`, so these could be added to different plots. What do you think?

**Comment by koehlerson** [comment]
ah so 

```julia
plot_grid(grid)
plot_nodenumbering!(grid)
```
?

**Comment by koehlerson** [comment]
btw we currently support **only** node numbering, not vertex numbering, see e.g this quadratic quadrilateral

![grafik](https://user-images.githubusercontent.com/27497290/139540677-4e693343-7ba7-4944-9bfa-ee6bda71c0d9.png)

**Comment by termi-official** [comment]
Correct, removed the checkmark again. Currently only the vertices/nodes in linear geometry works.

**Comment by koehlerson** [comment]
Why should nodes in the nonlinear deformed case not work for the nodes? The incompressible elasticity is quadratic displacement field with linear geometry and I just tested isoparametric serendipity in the plasticity example:

![grafik](https://user-images.githubusercontent.com/27497290/139542270-328c28a3-b6b0-4883-b24b-703e06fb692a.png)
plasticity with serendipity looks fine to me. edges etc. don't work

**Comment by termi-official** [comment]
In your example the grid is linearly deformed, so it works. But, as you wrote, chosing the deformation field and the grid nonlinear leads to a nonlinear representation whoch we cannot capture yet. For the first try we can go with just pulling the default interpolation.
